File: oop_grund/Inlämning/Airport/Airport/staff/Staff.cs
namespace Airport.staff
{
    public abstract class Staff : Human
    {
        private float salary;

        protected Staff(string name, int age) : base(name, age)
        {
            
        }

        public float GetSalary()
        {
            return salary;
        }

        public abstract void Work();


    }
}

File: oop_grund/Inlämning/Airport/src/main/java/with/william/airport/human/Staff.java
package with.william.airport.human;

import with.william.airport.util.Country;

import java.time.LocalDate;

public abstract class Staff extends Human {

    private float salary;

    protected Staff(String fullName, Country country, LocalDate dateOfBirth, float salary) {
        super(fullName, country, dateOfBirth);
        this.salary = salary;
    }

    public abstract void Work();

    public float getSalary() {
        return salary;
    }
}
